js-webapp/README.md

```diff
@@ -212,18 +248,14 @@ Visit [http://localhost:8000/](http://localhost:8000/) to see the result of the
 
 _The resulting page if everything works._
 
-**TODO** (Recap and announce what else is coming)
+Looking at the plot of the equation we expect the root to be `-1.00`, which is exactly what the HTML page in the browser is showing.
 
-The result of root finding was calculated using the C++ algorithm compiled to a WebAssembly module, executed by some
-JavaScript and rendered on a HTML page.
+So what did we do, we wrote a simple algorithm in C++, wrote Emscripten bindings, compiled a WebAssembly module and ran the algorithm in a web browser using some JavaScript to talk to the module.
```

Comment:
Can we rephrase it? It is difficult to read. May be it helps to split it to multiple sentences.

Reply:
rewrote to bullet list

ps. also shortened cli section.